addons/l10n_in/models/account_invoice.py

```diff
@@ -97,11 +97,10 @@ def _post(self, soft=True):
             """Check state is set in company/sub-unit"""
             company_unit_partner = move.journal_id.l10n_in_gstin_partner_id or move.journal_id.company_id
             if not company_unit_partner.state_id:
-                raise ValidationError(_(
-                    "State is missing from your company/unit %(company_name)s (%(company_id)s).\nFirst set state in your company/unit.",
-                    company_name=company_unit_partner.name,
-                    company_id=company_unit_partner.id
-                ))
+                raise ValidationError(_("The state is missing in company %s (%s).\n"
+                "You can quickly fix this problem by Going to App 'Settings' From the top menu 'General Setting'.\n"
```

Comment:
```suggestion
                "You can quickly fix this problem by going to App 'Settings' From the top menu 'General Settings'.\n"
```
I wonder if a RedirectWarning would not be overkill then?  (If people would have so much trouble with it then it would make sense to immediately go to the company settings)

Reply:
@jco-odoo  I have changed the validation error to the RedirectWarning which will redirect users to the company associated with that particular record of account_move.